src/bionics.cpp

```diff
@@ -14,38 +14,51 @@
 #include <algorithm> //std::min
 #include <sstream>
 
-#define BATTERY_AMOUNT 100 // How much batteries increase your power
+namespace {
+constexpr int BATTERY_AMOUNT = 100; // How much batteries increase your power
```

Comment:
Funnily the battery amount from the Power Storage Mk II (`bio_power_storage_mkII`) is taken from a literal value at the *one* place it's used. This `BATTERY_AMOUNT` is also only used once. Maybe it should go into the json data. At the very least it a makes no sense to use a named constant for this, but not for the other bionic.